2022-01-29 10:50:09,678 INFO    MainThread:29289 [wandb_setup.py:_flush():71] setting env: {}
2022-01-29 10:50:09,678 INFO    MainThread:29289 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-29 10:50:09,678 INFO    MainThread:29289 [wandb_init.py:_log_setup():371] Logging user logs to /mnt/disk1/duyna/FimiCalibIdea/wandb/run-20220129_105009-1tjx1nsd/logs/debug.log
2022-01-29 10:50:09,678 INFO    MainThread:29289 [wandb_init.py:_log_setup():372] Logging internal logs to /mnt/disk1/duyna/FimiCalibIdea/wandb/run-20220129_105009-1tjx1nsd/logs/debug-internal.log
2022-01-29 10:50:09,679 INFO    MainThread:29289 [wandb_init.py:init():404] calling init triggers
2022-01-29 10:50:09,680 INFO    MainThread:29289 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'metric': 'mse', 'early_stop': 50, 'ssa': False, 'train_type': 'train', 'name': 'Singlecalib', 'data_mean': array([14.78165346, 17.64072122]), 'data_std': array([3.5851945 , 3.57261641])}
2022-01-29 10:50:09,680 INFO    MainThread:29289 [wandb_init.py:init():460] starting backend
2022-01-29 10:50:09,680 INFO    MainThread:29289 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-29 10:50:09,757 INFO    MainThread:29289 [backend.py:ensure_launched():216] starting backend process...
2022-01-29 10:50:09,830 INFO    MainThread:29289 [backend.py:ensure_launched():221] started backend process with pid: 29574
2022-01-29 10:50:09,831 INFO    MainThread:29289 [wandb_init.py:init():469] backend started and connected
2022-01-29 10:50:09,838 INFO    MainThread:29289 [wandb_init.py:init():533] updated telemetry
2022-01-29 10:50:10,016 INFO    MainThread:29289 [wandb_init.py:init():563] communicating current version
2022-01-29 10:50:10,942 INFO    MainThread:29289 [wandb_init.py:init():568] got version response 
2022-01-29 10:50:10,942 INFO    MainThread:29289 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-29 10:50:11,355 INFO    MainThread:29289 [wandb_init.py:init():606] starting run threads in backend
2022-01-29 10:50:16,366 INFO    MainThread:29289 [wandb_run.py:_console_start():1810] atexit reg
2022-01-29 10:50:16,367 INFO    MainThread:29289 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-29 10:50:16,367 INFO    MainThread:29289 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-29 10:50:16,370 INFO    MainThread:29289 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-29 10:50:16,370 INFO    MainThread:29289 [wandb_init.py:init():633] run started, returning control to user process
2022-01-29 10:50:37,695 INFO    MainThread:29289 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-29 10:50:37,700 INFO    MainThread:29289 [wandb_run.py:_restore():1752] restore
2022-01-29 10:50:40,431 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 857
  total_bytes: 857
}

2022-01-29 10:50:41,081 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 857
  total_bytes: 857
}

2022-01-29 10:50:41,418 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 857
  total_bytes: 14984
}

2022-01-29 10:50:41,521 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 857
  total_bytes: 15690
}

2022-01-29 10:50:41,623 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 857
  total_bytes: 15690
}

2022-01-29 10:50:41,733 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 857
  total_bytes: 15690
}

2022-01-29 10:50:41,835 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7115
  total_bytes: 15690
}

2022-01-29 10:50:41,938 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15690
  total_bytes: 15690
}

2022-01-29 10:50:42,040 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15690
  total_bytes: 15690
}

2022-01-29 10:50:42,143 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15690
  total_bytes: 15690
}

2022-01-29 10:50:42,245 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15690
  total_bytes: 15690
}

2022-01-29 10:50:42,348 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15690
  total_bytes: 15690
}

2022-01-29 10:50:43,147 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15690
  total_bytes: 15690
}

2022-01-29 10:50:44,035 INFO    MainThread:29289 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15690
  total_bytes: 15690
}
local_info {
}

2022-01-29 10:50:45,186 INFO    MainThread:29289 [wandb_run.py:_append_history():2130] rendering history
2022-01-29 10:50:45,187 INFO    MainThread:29289 [wandb_run.py:_append_summary():2085] rendering summary
2022-01-29 10:50:45,187 INFO    MainThread:29289 [wandb_run.py:_append_files():2180] logging synced files
